dts/bindings/i2c/zephyr,i2c-emul-controller.yaml

```diff
@@ -10,6 +10,11 @@ include: i2c-controller.yaml
 properties:
   reg:
     required: true
+  target-pio-mode:
```

Comment:
`target-buffer-mode`

Reply:
Marking resolved to keep the conversation in one thread. This is modeled after the `i2c_ite_enhance` driver which uses `target-pio-mode`.